test/jdk/java/awt/event/ComponentEvent/TextAreaTextEventTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2007, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since this is a new test we dn't need 2007.

Reply:
I have added the original date based on the above comment that’s we get during the PR review below
https://github.com/openjdk/jdk/pull/7512#issuecomment-1075171066